playwright/test/helpers/search.ts

```diff
@@ -1,59 +1,56 @@
 import { Page } from 'playwright';
 import { expect } from '@playwright/test';
 import { isMobile } from '../contexts';
-import { formatFilterMobileButton } from '../selectors/search';
-
-export const worksSearchForm = '#search-searchbar';
 
 export const searchQueryAndSubmit = async (
   query: string,
   page: Page
 ): Promise<void> => {
-  await page.fill(worksSearchForm, query);
-  await page.press(worksSearchForm, 'Enter');
+  const searchBar = page.getByLabel('Search the catalogue');
+
+  await searchBar.fill(query);
+  await searchBar.press('Enter');
```

Comment:
page.fill and page.press are now discouraged in favour of locator.fill and locator.press
https://playwright.dev/docs/api/class-page#page-fill
https://playwright.dev/docs/api/class-page#page-press